packages/Search/src/details/DTK_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -258,38 +256,46 @@ void DistributedSearchTreeImpl<DeviceType>::queryDispatch(
     ////////////////////////////////////////////////////////////////////////////
 }
 
+// NOTE: for some reason Kokkos::BinSort::sort() is not const...
```

Comment:
I added the `const` specifier in `kokkos/algorithms/src/Kokkos_Sort.hpp` and it seemed to compile just fine.
It might be worth opening an issue on the Kokkos side.

Reply:
c.f. kokkos/kokkos#1308